### DISCUSSION TITLE
Calendar invite dowloads but does not open

### DISCUSSION THREAD

**Discussion by GoldenRatio48** [add2cal/add-to-calendar-button#506] [Q&A]
I am creating the code in the sandbox, copying it in a code module in my wordpress website builder (divi). I have installed the plugin and activated it.
When I test by pressing the add-to-calendar button. the calendar entry downloads but I can't open it. I am testing Outlook on Windows 11.
What am I doing wrong?

**Comment by jekuer** [comment]
Can you please share your code?
The one for the calendar.
The ics code (open the file in a text editor).
Ideally the HTML which is generated on your website (sometimes, other Plugins sneak in some bad code)

**Comment by GoldenRatio48** [comment]
Than you for the quick response.

This is what I generated in the sand box and copied in the website code module:
<add-to-calendar-button 
  name="Ship of Heroes Big Raid"
  description="Play with me!"
  startDate="2023-12-02"
  startTime="11:00"
  endDate="2023-12-02"
  endTime="12:00"
  timeZone="America/Indiana/Indianapolis"
  location="https://store.steampowered.com/app/1890100/Ship_of_Heroes/"
  options="'Apple','Google','iCal','Outlook.com','Yahoo','Microsoft365'"
  listStyle="overlay"
></add-to-calendar-button>

The ICS code from text editor:
BEGIN:VCALENDAR
VERSION:2.0
PRODID:-// https://add-to-calendar-pro.com // button v2.4.3 //EN
CALSCALE:GREGORIAN
METHOD:PUBLISH
BEGIN:VTIMEZONE
TZID:America/Indiana/Indianapolis
X-LIC-LOCATION:America/Indiana/Indianapolis
LAST-MODIFIED:20230517T170336Z
BEGIN:DAYLIGHT
TZNAME:EDT
TZOFFSETFROM:-0500
TZOFFSETTO:-0400
DTSTART:19700308T020000
RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
END:DAYLIGHT
BEGIN:STANDARD
TZNAME:EST
TZOFFSETFROM:-0400
TZOFFSETTO:-0500
DTSTART:19701101T020000
RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
END:STANDARD
END:VTIMEZONE
BEGIN:VEVENT
UID:2935285e-8b77-49fb-925f-753b4e56f0c8
DTSTAMP:20231201T162832Z
DTSTART;TZID=America/Indiana/Indianapolis:20231202T100000
DTEND;TZID=America/Indiana/Indianapolis:20231202T110000
SUMMARY:Sample Event
DESCRIPTION:Play with me!
X-ALT-DESC;FMTTYPE=text/html:
 <!DOCTYPE HTML PUBLIC ""-//W3C//DTD HTML 3.2//EN"">
 <HTML><BODY>
 Play with me!
 </BODY></HTML>
LOCATION:https://store.steampowered.com/app/1890100/Ship_of_Heroes/
SEQUENCE:0
STATUS:CONFIRMED
CREATED:20231201T162821Z
LAST-MODIFIED:20231201T162821Z
END:VEVENT
END:VCALENDAR

The code on the page, extracted from view-source:https://shipofheroes.com/add-to-calendar-test/
			
				<div class="et_pb_code_inner"><add-to-calendar-button 
  name="Ship of Heroes Big Raid"
  description="Play with me!"
  startDate="2023-12-02"
  startTime="11:00"
  endDate="2023-12-02"
  endTime="12:00"
  timeZone="America/Indiana/Indianapolis"
  location="https://store.steampowered.com/app/1890100/Ship_of_Heroes/"
  options="'Apple','Google','iCal','Outlook.com','Yahoo','Microsoft365'"
  listStyle="overlay"
></add-to-calendar-button></div>
			</div>

**Reply by jekuer** [reply]
Does it make a difference, when setting the timezone to EST5EDT or only EST?

**Comment by GoldenRatio48** [comment]
Let me see if I can figure out how to do this. I am not a coder.

 

Jocelyne McGeever

412-805-9251

 ***@***.***> ***@***.***

 <https://ShipofHeroes.com> https://ShipofHeroes.com

**Reply by jekuer** [reply]
Simply in the code you put into the shortcode, replace the America/Indiana/Indianapolis part at timeZone="America/Indiana/Indianapolis"

**Reply by GoldenRatio48** [reply]
OK, so I replace with what text?

**Reply by jekuer** [reply]
Test 1: "EST5EDT"
Test 2: "EST"

**Reply by GoldenRatio48** [reply]
Neither one worked

**Reply by GoldenRatio48** [reply]
Curiously, I was not able to open with Outlook, but it opened with Calendar.

**Reply by jekuer** [reply]
I just tried it on my side multiple times - works nicely with Outlook and all other calendar apps I tried.
How do you try to open it?
Does right-click -> "Open with Outlook" work?

**Reply by GoldenRatio48** [reply]
For those reading this, the problem solved itself. The same files that would not open yesterday opened without issue today. This is an excellent plugin and I look forward to using it.

**Comment by GoldenRatio48** [comment]
Thank you for following up. I have tried double clicking, selecting “open with Outlook”, and “open with Calendar”. Only the last one opened. But it is good to know that it is working for you. I should add that I am usually able to open .ics files.

 

It is really unfortunate because what you have is exactly what I need. I would gladly pay for the plugin. If I could make it work. 

 

Jocelyne O. McGeever

412-805-9251

 ***@***.***> ***@***.***

 <https://ShipofHeroes.com> https://ShipofHeroes.com

**Reply by jekuer** [reply]
Since you are able to open it at least in some way, while it is working completely fine on my side (with the same setup), most probably it is still some issue with your machine and/or Outlook settings. Are you using a very old version of Outlook? Or maybe you have some restrictions in place, limiting the ability to open files, which got downloaded from the web. Either way, that's hard to debug remotely, unfortunately.

**Comment by GoldenRatio48** [comment]
I understand. Every system is different. I am on the version of Outlook which came with Window 11, the most recent version, and as I said, I am able to open other ICS files from other sources. I understand that it is not reasonable to expect you to make your plugin work in all peculiar circumstances. But at the same time, you interact with a lot of users and you might have seen this issue before and know what is causing this. But I agree that it is likely something at my end. I just don’t know what it might be. I will investigate further and let you know what I find if I am able to solve it. Perhaps that will help you help someone else in the future.

 

Thank you for all your efforts and your wonderful plugin.

 

Jocelyne O. McGeever

412-805-9251

 ***@***.***> ***@***.***

 <https://ShipofHeroes.com> https://ShipofHeroes.com

**Reply by jekuer** [reply]
Do you maybe have one of those working files?
This could help to debug indirectly. Otherwise, it's impossible as I have no chance to reproduce this behavior.

**Comment by GoldenRatio48** [comment]
Now, this is very curious. Now I can open the files I downloaded yesterday without problem. And yet, I did not make any changes. Really strange.

 

I appreciate your trying to help me. You have a wonderful plugin. Thank you again.

 

Jocelyne O. McGeever

412-805-9251

 ***@***.***> ***@***.***

 <https://ShipofHeroes.com> https://ShipofHeroes.com
